Library/Homebrew/cmd/audit.rb

```diff
@@ -238,13 +238,41 @@ def audit_urls
       problem "Savannah homepages should be https:// links (URL is #{homepage})."
     end
 
+    if homepage =~ %r[^http://((?:trac|tools|www)\.)?ietf\.org]
+      problem "ietf homepages should be https:// links (URL is #{homepage})."
+    end
+
+    if homepage =~ %r[^http://((?:www)\.)?gnupg.org/]
+      problem "GnuPG homepages should be https:// links (URL is #{homepage})."
+    end
+
+    # Freedesktop is complicated to handle - It has SSL/TLS, but only on certain subdomains.
+    # To enable https Freedesktop change the url from http://project.freedesktop.org/wiki to
+    # https://wiki.freedesktop.org/project_name.
+    # "Software" is redirected to https://wiki.freedesktop.org/www/Software/project_name
+    # But is more complicated/tricky to get right, so don't enforce for now.
+    if homepage =~ %r[^http://((?:www|nice|libopenraw|liboil|telepathy|xorg)\.)?freedesktop\.org/(?:wiki/)?]
+      if homepage =~ /Software/
+        # Blank line to do absolutely nothing here.
+      else
```

Comment:
To be honest, I was almost deliberately leaving a notation for myself here, to come back to this particular line in future, because I thought the ` Software ` urls might cause a headache for PR creators, but I'm tempted to implement it and see how it goes. That's why I wanted to be more explicit about it rather than just silently excluding it as such.